project.godot

```diff
@@ -14,32 +14,116 @@ _global_script_classes=[ {
 "language": "GDScript",
 "path": "res://Assets/World/Buildings/Building.gd"
 }, {
+"base": "Spatial",
+"class": "Game",
+"language": "GDScript",
+"path": "res://Assets/World/Game.gd"
+}, {
+"base": "HBoxContainer",
+"class": "GameTypeButton",
+"language": "GDScript",
+"path": "res://Assets/UI/Scripts/GameTypeButton.gd"
+}, {
+"base": "Node",
+"class": "Global",
+"language": "GDScript",
+"path": "res://Assets/World/Global.gd"
+}, {
+"base": "Ship",
+"class": "Huker",
+"language": "GDScript",
+"path": "res://Assets/World/Units/Ships/Merchants/Huker/Huker.gd"
+}, {
+"base": "Control",
+"class": "MainMenuUI",
+"language": "GDScript",
+"path": "res://Assets/UI/Scripts/MainMenuUI.gd"
+}, {
+"base": "Ship",
+"class": "Pirate",
+"language": "GDScript",
+"path": "res://Assets/World/Units/Ships/Pirates/Pirate/Pirate.gd"
+}, {
+"base": "Control",
+"class": "Player",
+"language": "GDScript",
+"path": "res://Assets/Player/Player.gd"
+}, {
+"base": "MeshInstance",
+"class": "PlayerStart",
+"language": "GDScript",
+"path": "res://Assets/Player/PlayerStart.gd"
+}, {
 "base": "Building",
 "class": "Residence",
 "language": "GDScript",
 "path": "res://Assets/World/Buildings/Citizens/Residential/Residence.gd"
 }, {
+"base": "KinematicBody",
+"class": "SelectionUnit",
+"language": "GDScript",
+"path": "res://Sandbox/UnitSelection/SelectionUnit.gd"
```

Comment:
`SelectionUnit` and `Ship_19_08_28` disappear from this file when I open the project, since they reference a non-existant `Sandbox` folder.